scripts/security/phase4_enforcement_probe.py

```
@@ -0,0 +1,577 @@
+
+from __future__ import annotations
+
+import json
+import os
+import secrets
+import sys
+import uuid
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import Iterable
```

Ruff (0.15.0)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)